src/pepr/operator/index.ts

```diff
@@ -35,6 +42,8 @@ if (process.env.PEPR_WATCH_MODE === "true" || process.env.PEPR_MODE === "dev") {
   void initAPIServerCIDR();
 }
 
+void initAllNodesTarget();
```

Comment:
We should wrap this in the same conditional as the one above:
```suggestion
// Pre-populate the Node CIDR list since we are not persisting it
// Note ignore any errors since the watch will still be running hereafter
if (process.env.PEPR_WATCH_MODE === "true" || process.env.PEPR_MODE === "dev") {
  void initAllNodesTarget();
}
```

This ensures that this init only runs on the watcher pod (1/3 of the pepr pods) since it's only relevant there. The admission pods don't control network policies so they don't need this code to run.